### DISCUSSION TITLE
Not seeing the full UI in iOS 16.3.1 Safari

### DISCUSSION THREAD

**Discussion by Zephyr80085** [quoid/userscripts#466] [Q&A] [answered]
Hi! Thank you all for creating this. I use it for studying Japanese on WaniKani. I’ve read through the readme, but when I follow the same steps outlined there, I do not get a full UI in which I can remove Userscripts. I only see the small window in the attached screenshot. I can turn Userscripts on and off, but nothing I have tried takes me to a UI that looks like the one in the readme, or offers any functionality that would delete a script.

Am I missing something obvious?

Thanks
![96EBFA54-F287-46BD-AE26-D96B5AA1299B](https://user-images.githubusercontent.com/131229131/232964220-155775d3-094a-46b3-9b68-8ba4d3dc17a3.png)

**Comment by ACTCD** [comment, marked as answer]
Currently, you need to manually delete the corresponding file in the set save location, such as in the `Files` App.

But we have a PR https://github.com/quoid/userscripts/pull/447 already in progress that will bring a new detail view for the `popup`, which will include actions like delete.

![trash](https://user-images.githubusercontent.com/101378590/233063001-94f535b7-5ea3-423c-be5f-7caec91858a1.jpg)
